apache-mesos-mixin/README.md

```diff
@@ -0,0 +1,117 @@
+
+Logs from Apache Mesos are not enabled by default which will result in empty log panels if the following step is not performed. When running the Mesos master and agent process, specify the logs director using `--log_dir=` for both the master and agent. This path will differ depending on if you are running Linux or Windows but is required so that Promptail can ingest the logs.
+
+For master:
+`--log_dir=/var/log/mesos/master/` on Linux masters
+`–-log_dir=C:\Program Files\mesos\master\` on Windows masters
+
+For agents:
+`--log_dir=/var/log/mesos/agent/` on Linux agents
+`–-log_dir=C:\Program Files\mesos\agent\` on Windows agents
+
+Apache Mesos logs are enabled by default in the `config.libsonnet` and can be removed by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+In order for the selectors to properly work for master and agent logs ingested into your logs datasource, please also include the matching `job`, and `mesos_cluster` labels onto the [scrape_configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics.
+
+```yaml
+scrape_configs:
+  - job_name: integrations/apache-mesos
+    static_configs:
+      - targets:
+        - localhost
+        labels:
+          job: integrations/apache-mesos
+          mesos_cluster: '<your-cluster-name>'
+          __path__: /var/log/mesos/master/*.log*
+      - targets:
+        - localhost
+        labels:
+          job: integrations/apache-mesos
+          mesos_cluster: '<your-cluster-name>'
+          __path__: /var/log/mesos/agent/*.log*
+```
+
+![Apache Mesos Overview Dashboard 1](TODO)
+![Apache Mesos Overview Dashboard 2](TODO)
+
+## Alerts Overview
+
+| Alert                       | Summary                                                  |
+|-----------------------------|----------------------------------------------------------|
+| ApacheMesosHighMemoryUsage  | There is a high memory usage for the cluster.            |
+| ApacheMesosHighDiskUsage    | There is a high disk usage for the cluster.              |
+| ApacheMesosUnreachableTasks | There are an unusually high number of unreachable tasks. |
+| ApacheMesosNoLeaderElected  | There is currently no cluster coordinator.               |
+| ApacheMesosInactiveAgents   | There are currently inactive agent clients.              |
+
+Default thresholds can be configured in `config.libsonnet`
+
+```js
+{
+  _config+:: {
+    alertsWarningMemoryUsage: 90,
+    alertsCriticalDiskUsage: 90,
+    alertsWarningUnreachableTask: 3,
+  },
+}
+```
+
+## Install Tools
+
+```bash
+go install github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest
+go install github.com/monitoring-mixins/mixtool/cmd/mixtool@latest
+# or in brew: brew install go-jsonnet
```

Comment:
Minor nitpick here, this wording suggests go-jsonnet also includes mixtool, which isn't the case

Reply:
removed comment line and moved up the jsonnetfmt install.